File: Go/hello.go
/* Conceived by Sean Hinchee */
package main

import (
"fmt"
)

func main() {
	var a, b int
	usrname := ""

	fmt.Print("What's your name?: ")
	fmt.Scan(&usrname)
	fmt.Println("Hello, ", usrname, "!")

	a, b = 2, 2
	c := a + b
	fmt.Println("Two plus two is: ", c)
}

File: Ruby/hello.rb
# Conceived by Sean Hinchee

def main()
	print "What is your name?: "
	$usrname = $stdin.gets.chomp
	print "Hello, #{$usrname}!\n"

	$a = 2
	$b = 2
	$c = $a + $b
	print "2 plus 2 equals: #{$c}!\n"
end

main()
